src/scenic/syntax/ast.py

```diff
@@ -10,6 +10,57 @@ def __init__(self, *args: any, **kwargs: any) -> None:
 
 
 # special statements
+
+
+class ScenicTry(AST):
```

Comment:
Let's call this `ScenicTryInterrupt` since in your implementation it isn't in fact used for ordinary `try-except` statements.

Reply:
I changed it to `TryInterrupt` (see https://github.com/BerkeleyLearnVerify/Scenic/pull/79#discussion_r950783059)

The reason I called it `ScenicTry` originally was to disambiguate it from Python's `ast.Try` but since we're now calling it `TryInterrupt` I don't think we need to add `Scenic` at the beginning. Let me know if you think we still should have the prefix.